Realm/RLMObjectBase.mm

```diff
@@ -221,6 +221,16 @@ + (RLMObjectSchema *)sharedSchema {
     return [RLMSchema sharedSchemaForClass:self.class];
 }
 
++ (void)initializeLinkedObjectSchemas {
+    RLMObjectSchema *thisObjectSchema = [self sharedSchema];
+    for (RLMProperty *prop in thisObjectSchema.properties) {
```

Comment:
This is something that should be done as part of schema init rather than each time an object is created. `+[RLMSchema sharedSchemaForClass:]` is probably the appropriate place?